docs/guide/templates.md

```diff
@@ -181,14 +181,18 @@ end
 
 Code editors commonly add a trailing newline character to source files in keeping with the Unix standard. Including trailing whitespace in component templates can result in unwanted whitespace in the HTML, eg. if the component is rendered before the period at the end of a sentence.
 
-To strip trailing whitespace from component templates, use the `strip_trailing_whitespace` class method.
+To strip trailing whitespace from component templates, use the `strip_trailing_whitespace` component-local config option.
 
 ```ruby
 class MyComponent < ViewComponent::Base
   # do strip whitespace
-  strip_trailing_whitespace
+  configure_component do |config|
```

Comment:
Let's do:

```suggestion
  configure_view_component do |config|
```

As it matches our generator config change coming in `v4` (`generate component` -> `generate view_component`)